### ISSUE TITLE
"live update" does not "live update"

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#17542]
## Description
The "live update" option does not live update the selected maps.

### How to reproduce
check the "live update"-settings, nothing happens (beside update of displayed date and time here in this section, but not the map itself)
In the "maps > local"-section still is an old time/date displayed. Only tapping "update map" here in this section does update the map (downloading starts).
### Actual result
maps do still have an older date
### Expected result
updated maps
**Environment**
OsmAnd Version: +4.5.3
Android/iOS version: 13
Device model: S20+

Here you see, "live update" tells me, maps are up to date at today 9:XX (but still "orange")
![Screenshot_20230703_091600_OsmAnd+](https://github.com/osmandapp/OsmAnd/assets/95540936/5c136dde-3eaf-4866-ba31-e3236ec52474)

But in the "local" section:

Austria: date 01.06.23

![Screenshot_20230703_091102_OsmAnd+](https://github.com/osmandapp/OsmAnd/assets/95540936/8d2c6d69-adc0-4f51-9dff-bedaf746df64)

Greece: date 01.06.23

![Screenshot_20230703_091112_OsmAnd+](https://github.com/osmandapp/OsmAnd/assets/95540936/6bef8b0e-e6d5-4c45-876c-8da6404c304b)

Only tapping "update map" here in the local section updates the map.

**Comment by scaidermern** [comment]
I guess this is the expected behavior. "Live update" downloads the diff between the base map and the current OSM database. The local tab allows to update the base map.

However, the current behavior can indeed be improved. The live update tab should provide a clear hint to the user that the base map is out of date. How about adding such a notification?

**Comment by DmitryAlexei** [comment]
You can read about live updates here: https://osmand.net/docs/user/personal/maps#osmand-live
OsmAnd Live is a special feature that provides additional **small map updates on top of the large complete monthly maps**.
"Big maps" need to be updated manually or using **Update all** button  on the bottom of the *Updates screen*.
<img width="249" alt="Screenshot_35" src="https://github.com/osmandapp/OsmAnd/assets/116079541/7a6e4b17-4a6a-401e-ba32-8862d2336916">

**Comment by Rowin63** [comment]
Yes, I know about the "small parts" and about the "update all"-button, thanks ;-)
But I was also expecting a "green" map symbol ("hello, I´m up to date") also after a live update.
So to me it is/was not clear, if "up to date" or not.

**Comment by Korb** [comment]
A related question about the non-obvious UI: is the `Latest OpenStreetMap update available: 2026-05-27 15:50` text a **button**? I thought so, but when I click it and have internet access, nothing happens, the notification drawer doesn't show the download progress, and the text on the page doesn't change.

<img width="356" height="502" alt="Image" src="https://github.com/user-attachments/assets/010d9bf1-137f-429b-833b-ed59bca209b2" />

### Environment
OsmAnd: 5.3.10 (531003) beta f-droid (2026 -05-11)
Android/iOS: 14 (Funtouch OS 14)
Device: Vivo T1 (4G) / V2153
